<?xml version="1.0"?>
<launch>
      <param name="use_sim_time" value="false"/>
      <arg name="drone_name" default="iris"/>
      <arg name="vmax" default="0"/>
      <arg name="rmax" default="2"/>
      <arg name="x_offset" default="0"/>
      <arg name="update_freq" default="10"/>
      
      <group ns="$(arg drone_name)">
        <node name="vicon_publish_mp_setpoints" pkg="training_q_learning" type="vicon_publish_mp_setpoints.py" respawn="true" output="screen">
        <param name="drone_name" value="$(arg drone_name)"/>
        <param name="vmax" value="$(arg vmax)"/>
        <param name="rmax" value="$(arg rmax)"/>
        <param name="x_offset" value="$(arg x_offset)"/>
        <param name="update_freq" value="$(arg update_freq)"/>
        </node>
      </group>  
</launch>